custom_reports/object_management_api/definitions/custom_dimension.yaml

```diff
@@ -19,7 +20,16 @@ properties:
       - name
       - active
       - case_sensitive
+      - scope
+      - website_id
+      - slot
```

Comment:
slot here, but is deleted below. 

It is read only so probably should be present, but definitely not required.